addons/event/views/event_registration_views.xml

```diff
@@ -69,11 +69,11 @@
                             <field class="text-break" name="event_id"
                                    context="{'name_with_seats_availability': True}" options="{'no_create': True}"/>
                             <field name="slot_id" invisible="not is_multi_slots" required="is_multi_slots"
-                                   options="{'no_create': True}"
-                                   domain="[('is_sold_out', '=', False)]"/>
+                                   context="{'name_with_seats_availability': True, 'availability_for_ticket': event_ticket_id}"
```

Comment:
Shouldn't we display the slot_ticket instead of hacking slot_id name ? Have to say number of computed seats is definitively under no-control ^^ fearing we enter too much computation and displaying too much stuff.